src/universal/modules/teamDashboard/components/TaskColumn/TaskColumnDropZone.tsx

```diff
@@ -1,13 +1,11 @@
-import {DropTarget} from 'react-dnd'
 import React, {ReactElement} from 'react'
+import {DropTarget} from 'react-dnd'
 
 import withAtmosphere from 'universal/decorators/withAtmosphere/withAtmosphere'
 import sortOrderBetween from 'universal/dnd/sortOrderBetween'
 import UpdateTaskMutation from 'universal/mutations/UpdateTaskMutation'
+import {ITask, IUpdateTaskInput, TaskStatusEnum} from 'universal/types/graphql'
```

Comment:
What is the leading I convention? interface? (eg. `ITask`)

Reply:
Probably answered my own question: `export interface ITeam {` from `graphql.d.ts`